buildroot-external/board/raspberrypi/yellow/uboot.config

```diff
@@ -1,5 +1,22 @@
+CONFIG_TARGET_RPI_4=y
+CONFIG_BCM2712=y
+
+CONFIG_MMC_SDHCI_BCMSTB=y
+
+CONFIG_DM_USB_GADGET=y
+
+CONFIG_USB_GADGET=y
+CONFIG_USB_GADGET_MANUFACTURER="FSL"
+CONFIG_USB_GADGET_VENDOR_NUM=0x0525
+CONFIG_USB_GADGET_PRODUCT_NUM=0xa4a5
+CONFIG_USB_GADGET_DWC2_OTG=y
+CONFIG_USB_GADGET_DOWNLOAD=y
```

Comment:
Is this working? If not, maybe remove for now? :thinking:

Reply:
It works on CM4, so we should keep it enabled.